engine/apps/api/views/alert_group.py

```diff
@@ -558,7 +578,7 @@ def filters(self, request):
                 "name": "status",
                 "type": "options",
                 "options": [
-                    {"display_name": "new", "value": AlertGroup.NEW},
+                    {"display_name": "firing", "value": AlertGroup.NEW},
```

Comment:
could this be done in a separate PR? Are you sure to have updated all other references to "new" to "firing"? If we make this change I think it'd also make sense to refactor `AlertGroup.NEW` -> `AlertGroup.FIRING`

Reply:
I made alert groups page look consistent, also it is called "firing" everywhere in the docs, we can finish that renaming in the separate PR

<img width="837" alt="Screenshot 2023-03-20 at 12 40 46" src="https://user-images.githubusercontent.com/2262529/226248725-0ef994ee-14c6-4d3f-88c9-a403006ad806.png">